card/detekt-baseline.xml

```diff
@@ -3,5 +3,6 @@
   <ManuallySuppressedIssues/>
   <CurrentIssues>
     <ID>MemberNameEqualsClassName:ThreeDSecureWebViewClientTest.kt$ThreeDSecureWebViewClientTest$@Test fun threeDSecureWebViewClientTest()</ID>
+    <ID>NotNullAssertionOperatorRule:CardComponentProviderTest.kt$CardComponentProviderTest$paymentMethodType!!</ID>
```

Comment:
You should just be able to add `@Suppress("NotNullAssertionOperatorRule")` to the test file itself (or method, but file is fine since it's a test.

Reply:
yep thanks, that's a better idea